test/library/packages/Shlex/Test.chpl

```diff
@@ -0,0 +1,167 @@
+use Shlex;
```

Comment:
Are you familiar with Chapel's [testing system](https://chapel-lang.org/docs/master/developer/bestPractices/TestSystem.html#readme-testsystem)? Basically, you'll want to add a `Test.good` file that contains the expected output from this file. Also, maybe we can name these something like `ShlexTests.chpl` and `ShlexTests.good`? 

You can optionally split these tests up across multiple files for finer granularity testing as well as the ability to convert tests for features that not yet implemented into futures.